client/app/store.js

```diff
@@ -2,11 +2,13 @@ import { configureStore } from '@reduxjs/toolkit';
 import logger from 'redux-logger';
 import authReducer from '../features/auth/authSlice';
 import productReducer from '../features/products/productSlice';
```

Comment:
Does this need to be productsReducer (with an s) since the slice is named "products"? I forget.